pilz_trajectory_generation/src/joint_limits_container.cpp

```diff
@@ -20,141 +20,75 @@
 #include "ros/ros.h"
 #include <stdexcept>
 
-bool pilz::JointLimitsContainer::addLimit(const std::string &joint_name, pilz_extensions::JointLimit joint_limit)
+namespace pilz
+{
+
+bool JointLimitsContainer::addLimit(const std::string &joint_name, pilz_extensions::JointLimit joint_limit)
 {
   if(joint_limit.has_deceleration_limits && joint_limit.max_deceleration >= 0)
   {
     ROS_ERROR_STREAM("joint_limit.max_deceleration MUST be negative!");
     return false;
   }
-  else
+  if (!container_.insert(std::pair<std::string, pilz_extensions::JointLimit>(joint_name, joint_limit)).second)
```

Comment:
It's not a good style (with regard to readability) to perform more than one operation in a one line.

Reply:
Done. Please review changes